mw/global.d.ts

```diff
@@ -48,7 +48,7 @@ declare global {
      * @return {HTMLLinkElement} Link tag
      * @see https://doc.wikimedia.org/mediawiki-core/master/js/#!/api/global-method-importStylesheet
      */
-    function importStylesheet(title: string): HTMLLinkElement | null;
+    function importStylesheet(title: string): HTMLLinkElement;
```

Comment:
`importStylesheet` always create a new <link>, unlike `importScript` it does not check for duplicates.